x-pack/plugins/infra/public/alerting/metric_threshold/hooks/use_metrics_explorer_chart_data.ts

```diff
@@ -55,3 +70,19 @@ export const useMetricsExplorerChartData = (
     null
   );
 };
+
+const mapMetricThresholdMetricToMetricsExplorerMetric = (metric: MetricExpressionCustomMetric) => {
+  if (metric.aggType === 'count') {
+    return {
+      name: metric.name,
+      aggregation: metric.aggType as MetricExplorerCustomMetricAggregations,
```

Comment:
I changed it from my end to be `'count' as  as MetricExplorerCustomMetricAggregations` since Typescript seems to be unable to decipher `count` as part of `MetricExplorerCustomMetricAggregations` even though it clearly is.